lib/ansible/parsing/dataloader.py

```diff
@@ -233,35 +236,33 @@ def path_dwim(self, given):
         given = unquote(given)
         given = to_text(given, errors='surrogate_or_strict')
 
-        if given.startswith(u"/"):
-            return os.path.abspath(given)
-        elif given.startswith(u"~"):
-            return os.path.abspath(os.path.expanduser(given))
+        if RE_ABS_PATH.search(given):
```

Comment:
restored for the abs path, but kept tasks, that is waaay too many ifs to get same filter